Makefile

```diff
@@ -33,6 +39,11 @@ IMPT_IMG_NAME=cdi-$(IMPORTER)
 GIT_USER=$(shell git config --get user.email | sed 's/@.*//')
 TAG=$(GIT_USER)-latest
 
+# Preflight Check Defaults
+USE_DOCKER=1
+$(eval DOCKER_OUT=$(shell docker ps))
```

Comment:
This mixes make syntax (everything left justified) and bash syntax (everything under a target indented by 1).

To set the variable globally in the Makefile, you can do
DOCKER_OUT=$(shell docker ps)
DOCKER_OUT != docker ps